apps/web/tests/asset-inventory/collector/collector-details/edit-collector.spec.ts

```diff
@@ -27,37 +27,40 @@ test.describe('Edit Schedule', () => {
 
         await expect(pageTitleName).toContainText('Collector');
     };
+    const clickCollector = async (collector) => {
+        // Click collector.
+        currentCollectorName = await collector.textContent();
+        await collector.click();
+
+        // Wait for navigation to the collector details page.
+        await page.waitForNavigation();
```

Comment:
suggestion: I have confirmed that `waitForNavigation` is soon going to be deprecated. Instead, it is recommended to use `waitForURL`!
https://playwright.dev/docs/api/class-page#page-wait-for-navigation

Reply:
![image](https://github.com/cloudforet-io/console/assets/50662170/45560104-d073-40b1-99c8-7d676f0f60a1)